.pullapprove.yml

```diff
@@ -1,45 +1,44 @@
-# You *must* specify "version: 2"
-version: 2
-
-requirements:
-  signed_off_by:
-    required: false
-
-always_pending:  # A pull request matching any of these conditions will always have a "pending" status for this group.
-  title_regex: '(WIP|wip)'
-  explanation: 'Work in progress...'
-
-# Group settings to apply to all groups by default, optionally being overridden later
-group_defaults:
-  reset_on_push: # Whether or not approval statuses get reset to "pending" when new commits are pushed to a PR
-    enabled: false # Set to false for efficiency
-  reset_on_reopened: # Whether or not approval statuses get reset to "pending" when a pull request is re-opened
-    enabled: true
-  author_approval: # Whether or not the author must approve the pull request.
-    required: false
-    ignored: true
-  approve_by_comment:
-    enabled: false
-  reject_value: -1 # Essentially allows rejections to cancel out approvals
-
-# Groups of reviewers and their respective settings
+version: 3
+
 groups:
-  computational-biologists:
-    required: 1 # Require approval from at least one computational biologist
+  pullapprove-admins:
     conditions:
-      labels:
-        exclude:
-          - ENG-ONLY
-    users:
-      - ambrosejcarr
-      - jishuxu
-      - mckinsel
-      - mbabadi
-      - kishorikonwar
-
-  software-engineers:
-    required: 1 # Require approval from at least one software engineer
-    users:
-      - dshiga
-      - samanehsan
-      - rexwangcc
+    - "'.pullapprove.yml' in files"
+
+    reviews:
+      required: 1
+      author_value: 1 # if author in group reduce number of approvals needed by this number
+
+    reviewers:
+      users:
+      - rhiananthony
+      teams:
+      - secops
+
+  software-review:
+    reviews:
+      required: 2
```

Comment:
Why are two software reviews required? I thought it was one software and one computational review

Reply:
This requires 2 software reviews unless the author is part of that group, then it only requires one. For instance, if anyone on hornet creates a PR then it only requires one more SE and one CB approval. If someone else outside of this group decided to create a PR - Sam for instance, then it would require review by 2 SEs and one CB. Does that make sense? I can make it more simple, but was experimenting :)

Reply:
Oh I see! I vote keeping it one SE and one CB approval even for external changes to skylab

Reply:
Done :)